main.go

```diff
@@ -333,9 +333,9 @@ func run(args []string) error {
 	go pClient.UpdatePrometheusMetrics()
 
 	// Read CA bundle for passing to metrics library
-	ca, err := caBundle(*caBundlePath)
+	ca, err := certloader.LoadTrustStore(*caBundlePath)
```

Comment:
If I'm reading this right, then metrics endpoint will not reload the CA bundle. Should this be documented?

Reply:
Yeah -- we should. It's a bit of an icky limitation that I'm not sure how to solve right now.